src/mandelbrot.rs

```diff
@@ -0,0 +1,104 @@
+// The Computer Language Benchmarks Game
+// http://benchmarksgame.alioth.debian.org/
+//
+// contributed by Matt Watson
+// contributed by TeXitoi
+
+use std::io::Write;
+use std::thread;
+const THREADS: usize = 20;
+const MAX_ITER: usize = 50;
+const VLEN: usize = 8;
+const ZEROS: Vecf64 = [0.0; VLEN];
+
+pub type Vecf64 = [f64; VLEN];
```

Comment:
If you want to play with rust operator overloading, you can define a tuple struct: `pub struct Vecf64([f64; VLEN]);` and then implement the Add, Mul and Sub traits on it.